README.md

```diff
@@ -1,11 +1,64 @@
 # Description
 Coforma's response to the SWIFT technical challenge
 
-## Local Setup
+## TODO: Table of Contents
+
+- TODO: Links here
+
+## Getting Started
+
+### One time only
+
+Before developing locally, you'll need to install some tools. You may have some of these already, or you may not.
+
+- Install [nvm](https://github.com/nvm-sh/nvm):
+    ```bash
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```

Comment:
Curious why you did not use brew for this one like the others

Reply:
This is what nvm recommends for installation. Otherwise you can get some weird versioning issues. 🤷